### PR TITLE
Adjust the containerized proxy to match the new installation process

### PR DESCRIPTION
**Pull request by cbosdo**
# Description

Since 2022.08 and 4.3.1, the containerized proxy installation and configuration has slightly changed. This PR updates the doc according to the new standard.

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [X] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by jcayouette — APPROVED** [review]

File: modules/installation-and-upgrade/pages/proxy-container-installation.adoc [hunk 1]
```diff
@@ -93,21 +93,23 @@ zypper install uyuni-proxy-systemd-services
...
 
-- [path]``/var/lib/uyuni/proxy-squid-cache``
-- [path]``/var/lib/uyuni/proxy-rhn-cache``
-- [path]``/var/lib/uyuni/proxy-tftpboot``
+- [path]``uyuni-proxy-squid-cache``
+- [path]``uyuni-proxy-rhn-cache``
+- [path]``uyuni-proxy-tftpboot``
 
-To modify location of those paths, edit [path]``/etc/sysconfig/uyuni-proxy-systemd-services.config``.
+To override the default volume settings, create the volumes prior to the first start of the pod using the [literal]``podman volume create`` command
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-installation.adoc:104]:
```suggestion
To override the default volume settings, create the volumes prior to the first start of the pod using the [literal]``podman volume create`` command.
```

File: modules/installation-and-upgrade/pages/proxy-container-installation.adoc [hunk 2]
```diff
@@ -93,21 +93,23 @@ zypper install uyuni-proxy-systemd-services
...
 
-In this file it is possible to add custom arguments passed to podman container pod:
+In the [path]``/etc/sysconfig/uyuni-proxy-systemd-services.config`` file it is possible to add custom arguments passed to podman container pod:
 
 - [literal]``EXTRA_POD_ARGS=''``
 
-In this file it is possible to modify path where proxy configuration is expected, however modifying it is strongly discouraged:
+In this file it is possible to modify the tag to use for the container images
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-installation.adoc:110]:
```suggestion
In this file it is possible to modify the tag to use for the container images:
```

File: modules/installation-and-upgrade/pages/proxy-container-installation.adoc [hunk 3]
```diff
@@ -93,21 +93,23 @@ zypper install uyuni-proxy-systemd-services
...
 
-- [literal]`CONFIG_DIR`=[path]`/etc/uyuni/proxy`
+- [literal]`TAG`=[path]`latest`
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-installation.adoc:112]:
```suggestion
- [literal]``TAG``=[path]``latest``
```

**Review by keichwa — CHANGES_REQUESTED** [review]
Some cosmetic changes.

**Comment by cbosdo** [comment]
@keichwa I just merged all your suggestions

**Review by keichwa — APPROVED** [review]
Thank you, Cedric!  If you don't veto, I'll merge it in the afternoon.

**Pull request merged**